rabbitmq/tests/test_e2e.py

```diff
@@ -5,28 +5,25 @@
 import logging
 
 import pytest
-from packaging import version
 
 from datadog_checks.dev.utils import get_metadata_metrics
 
-from .common import CONFIG, OPENMETRICS_CONFIG, RABBITMQ_VERSION
+from .common import CONFIG, METRICS_PLUGIN, OPENMETRICS_CONFIG
 from .metrics import DEFAULT_OPENMETRICS, FLAKY_E2E_METRICS, assert_metric_covered
 
 log = logging.getLogger(__file__)
 
 pytestmark = [pytest.mark.e2e]
 
 
-@pytest.mark.skipif(RABBITMQ_VERSION >= version.parse("3.8"), reason="Test legacy check on rabbitmq versions < 3.8")
-def test_rabbitmq_e2e(dd_agent_check):
+@pytest.mark.skipif(METRICS_PLUGIN == "prometheus", reason="Not testing management plugin metrics.")
```

Comment:
nit - you could add to the conftest file: 

```
requires_prometheus = pytest.mark.skipif(METRICS_PLUGIN == "prometheus", reason="Not testing management plugin metrics.")
```

And use `@requires_prometheus` everywhere instead of copy/paste them. Same for `management`